packages/chat-component/src/core/parser/index.ts

```diff
@@ -52,9 +52,10 @@ export async function parseStreamedMessages({
     // followup questions are marked either with the word 'Next Questions:' or '<<text>>' or both at the same time
     // these markers may be split across multiple chunks, so we need to buffer them!
     // TODO: remove all this logic from the frontend and implement a solution on the backend or with TypeChat
-
+    // we start by creating a buffer when we match the first marker
     const matchedFollowupQuestionMarker =
       (!isFollowupQuestion && chunkValue.includes('Next')) || chunkValue.includes('<<');
```

Comment:
btw should we still parse `Next` markers? Is the new model still using `Next` to format followup questions?

Reply:
If the updated model uses only `<<` and `>>` to format followup questions, then I'd suggest changing the condition to:

`const matchedFollowupQuestionMarker = !isFollowupQuestion && chunkValue.includes('<<');`

Reply:
Good question. @sinedied  can you clarify?